### ISSUE TITLE
Go to page where certain row is present

### ISSUE THREAD

**Issue by 57even** [aslagle/reactive-table#287]
If I have `_id` or something else of a row in a collection, can I programatically go to the page in the reactive table where the row is present?

The situation is that I have clicked a row and saved its ID but the user can still browse through the many pages and will now go back to the page at which the clicked row is present.

**Comment by aslagle** [comment]
Yes, but it's a little complicated to set up. You can set up a ReactiveVar that reactive table will use for the current page, and then get and set it outside of the table. See https://github.com/aslagle/reactive-table#accessing-and-controlling-table-state
